src/integration-test/resources/StatusIT/__files/test01_getStatuses/request.json

```diff
@@ -0,0 +1,7 @@
+{
+	"letterIds": [
+		"43a32404-28ee-480f-a095-00d48109afab",
```

Comment:
Maybe include an id that has a present status in the db, for example 0d458afc-526f-4d1d-aa0c-5a7228c37382?